app/application/services/auth/signup_service.py

```
@@ -0,0 +1,33 @@
+from app.application.services.auth.jwt_service import JWTService
+from app.interface.schemas.auth.requests import SignUpRequest
+from app.infrastructure.repositories.auth import AuthRepository
+from app.infrastructure.models.auth import User
+from app.infrastructure.exceptions import DatabaseIntegrityException
+
+
+class SignUpService:
+    def __init__(
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)